unetsocket/js/test/spec/unet.spec.js

```diff
@@ -195,3 +195,36 @@ describe('A UnetSocket', function () {
 
 });
 
+describe('Unet Utils', function () {
+  function coordTester(res, val) {
+    const tolerance = 0.000100;
+    const min = res - tolerance;
+    const max = parseFloat(res) + tolerance;
+    return (val >= min && val <=max);
+  }
+
+  beforeEach(function() {
+    jasmine.addCustomEqualityTester(coordTester);
+  });
+
+  it('should be able to convert local coordinates to GPS coordinates', function () {
+    const origin=[1.34286,103.84109], x=100, y=100;
+    let loc = toGps(origin,x,y);
+    expect(loc.length).toBe(2);
+    expect(loc[0]).toBeInstanceOf(Number);
+    expect(loc[1]).toBeInstanceOf(Number);
+    expect(loc[0]).toEqual(1.343764);
+    expect(loc[1]).toEqual(103.841988);
+  });
+
+  it('should be able to convert GPS coordinates to local coordinates', function () {
+    const origin=[1.34286,103.84109], lat=1.343764, lon=103.841988;
```

Comment:
Can we add one more point to check for each x,y => lat,lon and lat,lon => x,y conversions? So total 2 points. Since we're doing 2D math, it's safer to check against 2 different points.

Reply:
Added another point for testing.